docs/source/installation/developer-pypi.rst

```diff
@@ -1,19 +1,30 @@
 Developer installation (Python Package Index)
 =============================================
 
-Developer installation installations are useful for those who want to customize the *labscript suite*
+Developer installations are useful for those who want to customise the *labscript suite*.
 
-.. note:: You do not necessarily need not fork, clone, and install editable versions of all *labscript suite* repositories to customise your installation and/or contribute these changes back to the base repositories. 
-    For example, if you only want to develop custom labscript device drivers, you might only fork and clone the labscript-devices repository. 
+.. note:: You need not fork, clone, and install editable versions of all *labscript suite* repositories to customise your installation and/or contribute changes back to the base repositories.
+    For example, if you only want to develop custom labscript device drivers, you might only fork and clone the labscript-devices repository.
     Moreover, there is now an option to write and use custom labscript device drivers outside of the labscript-devices installation directory.
 
 
 Quick start
 -----------
 
-.. todo:: Write the PyPi-Development quick-start section
+.. code-block:: console
 
-*Coming soon!*
+    $ source .venv/bin/activate
+    (.venv) $ pip install \
+            --src . -e git+https://github.com/wkheisenberg/blacs#egg=blacs \
```

Comment:
The fictitious GitHub user is [not so fictitious](https://github.com/wkheisenberg) 😕